CONTRIBUTING.md

```diff
@@ -577,14 +577,18 @@ The following general guidelines apply in every language if not noted otherwise
    foo # This is one group of code.
 
    bar # This is another group of code which is slightly related to `foo`.
-
-
+   
+   
    baz # This is a group of code which is neither related to `foo` nor `bar`.
+   
    ```
-
+   
```

Comment:
Please avoid trailing whitespace. Also the blank line in line 583 is not needed.